src/lib/mgba/__tests__/websocket-connection-fix.test.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-condition */
```

Comment:
combine all these websocket-*.test.ts files into 1-2. reduce the number of tests and remove low quality or importance tests